instill/clients/model.py

```diff
@@ -124,12 +124,36 @@ def watch_model(
             metadata=self.hosts[self.instance].metadata,
         ).send_sync()
 
+    @grpc_handler
+    def watch_latest_model(
+        self,
+        model_name: str,
+        async_enabled: bool = False,
+    ) -> model_interface.WatchUserLatestModelResponse:
+        if async_enabled:
+            return RequestFactory(
+                method=self.hosts[self.instance].async_client.WatchUserLatestModel,
+                request=model_interface.WatchUserLatestModelRequest(
+                    name=f"{self.namespace}/models/{model_name}",
+                ),
+                metadata=self.hosts[self.instance].metadata,
+            ).send_async()
+
+        return RequestFactory(
+            method=self.hosts[self.instance].client.WatchUserLatestModel,
+            request=model_interface.WatchUserLatestModelRequest(
+                name=f"{self.namespace}/models/{model_name}",
+            ),
+            metadata=self.hosts[self.instance].metadata,
+        ).send_sync()
+
     @grpc_handler
     def create_model(
         self,
         name: str,
         definition: str,
         configuration: dict,
+        # todo: add uid, hardware?
```

Comment:
we have two new [required properties](https://github.com/instill-ai/protobufs/blob/main/model/model/v1alpha/model.proto#L190-L195) for model

Reply:
updated

Reply:
updated

Reply:
task type is wrong, its
```
task: common.task.v1alpha.task_pb2.Task.ValueType
```

two options here, you either has to do
```
import instill.protogen.common.task.v1alpha.task_pb2 as task_interface
.
.
.
@grpc_handler
    def create_model(
        self,
        name: str,
        task: task_interface.Task,
```
or do internal conversion from `str` to `task` in `create_model` method

Reply:
task type fixed